src/Account.js

```diff
@@ -84,6 +141,7 @@ export default class Account {
 
     return new Promise((resolve, reject) => {
       if(!receipt.status) {
+        logger.error('Transaction failed');
```

Comment:
Lets add more context such as printing the receipt object itself. Will make it easier to troubleshoot

Reply:
@raamb addressed this in [commit](https://github.com/singnet/snet-sdk-js/pull/3/commits/338443f13682ebe1872121e0235c40839b5fd26a) and [commit](https://github.com/singnet/snet-sdk-js/pull/3/commits/c45e1d2fe2c94fedb4e1c867cdc3fd6b8e7807a9)